package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "json-schema-bundle",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/zicht/json-schema-bundle.git"
+    },
```

Comment:
Dit is de repository van de zicht/framework-extra-bundle... Hoe zit dit dan?

Reply:
Ik wilde eerst een losse bundel maken, later besloten het in deze bestaande bundel toe te voegen.  Dit moet hernoemd worden.